docs/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thank you for your pull request!
+
+Please see https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md for guidelines on
+how to best contribute to the Windows App SDK Samples!
+
+-->
+
+##### Checklist
+
+- [ ] Required: Please ensure youre samples can be correctly built using Visual Studio 2019 and above
+- [ ] If your sample is related to one of the existing scenarios (ResourceManagement, Windowing, etc) 
```

Comment:
Not sure if we need this checklist point, since it's covered in docs/guidelines.md?

Reply:
Maybe I can make it clearer that this point is if the sample(s) are new :)